packages/flutter_tools/test/general.shard/android/android_studio_test.dart

```diff
@@ -486,6 +499,84 @@ void main() {
       Platform: () => platform,
       PlistParser: () => plistUtils,
     });
+
+    testUsingContext('Can find Android Studio 2020.3 bundled Java version on Mac', () {
+      final String studioInApplicationPlistFolder = globals.fs.path.join(
+        '/',
+        'Application',
+        'Android Studio.app',
+        'Contents',
+      );
+      globals.fs.directory(studioInApplicationPlistFolder).createSync(recursive: true);
+
+      final String plistFilePath = globals.fs.path.join(studioInApplicationPlistFolder, 'Info.plist');
+      plistUtils.fileContents[plistFilePath] = macStudioInfoPlist2020_3;
+      processManager.addCommand(FakeCommand(
+        command: <String>[
+          globals.fs.path.join(studioInApplicationPlistFolder, 'jre', 'Contents', 'Home', 'bin', 'java'),
+          '-version',
+        ],
+        stderr: '123',
+      )
+      );
+      final AndroidStudio studio = AndroidStudio.fromMacOSBundle(
+        globals.fs.directory(studioInApplicationPlistFolder).parent.path,
+      )!;
+
+      expect(studio.javaPath, equals(globals.fs.path.join(
+        studioInApplicationPlistFolder,
+        'jre',
+        'Contents',
+        'Home',
+      )));
+    }, overrides: <Type, Generator>{
+      FileSystem: () => fileSystem,
+      FileSystemUtils: () => fsUtils,
+      ProcessManager: () => processManager,
+      // Custom home paths are not supported on macOS nor Windows yet,
+      // so we force the platform to fake Linux here.
+      Platform: () => platform,
+      PlistParser: () => plistUtils,
+    });
+
+    testUsingContext('Can find Android Studio >=2022.1 bundled Java version on Mac', () {
```

Comment:
Putting a >= will eventually become outdated :)

```suggestion
    testUsingContext('Can find Android Studio 2022.1 bundled Java version on Mac', () {
```

Reply:
You are right, thanks for your suggestion.

Reply:
This applies to Windows and Linux tests as well.